.github/workflows/test.yml

```diff
@@ -230,6 +230,52 @@ jobs:
         # which pixi should be absolute
         which pixi | grep -q "^/"
 
+  existing-pixi-bin:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Move pixi.toml
+        run: mv test/default/* .
+      - name: Download pixi binary
+        run: |
+          set -o pipefail
+          mkdir -p ~/custom-existing-pixi
+          curl -L --output ~/custom-existing-pixi/pixi \
+            https://github.com/prefix-dev/pixi/releases/download/v0.14.0/pixi-x86_64-unknown-linux-musl
+          chmod +x ~/custom-existing-pixi/pixi
+          echo ~/custom-existing-pixi >> $GITHUB_PATH
+      - uses: ./
+        with:
+          cache: false
+      - run: |
+          set -o pipefail
+          pixi --version | grep -q "pixi 0.14.0"
+          which pixi | grep -q ~/custom-existing-pixi/pixi
+
+  dont-use-existing-pixi-bin:
```

Comment:
Checking if this works on windows would be nice.

Reply:
Done